responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -5,6 +5,7 @@ channels:
   - anaconda
 dependencies:
   - python=3.8
+  - pip
```

Comment:
is this needed?

Reply:
we were pinning pip to an older version before that I think contributed to conda grabbing some older versions of base python packages, along with the conda channels.

Right now in build log conda actually complain about missing pip:

Retrieving notices: ...working... done
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Collecting package metadata (repodata.json): ...working...